src/masternodes/rpc_vault.cpp

```diff
@@ -377,7 +378,7 @@ UniValue listvaults(const JSONRPCRequest& request) {
                         {
                             {
                                 "ownerAddress", RPCArg::Type::STR, RPCArg::Optional::OMITTED,
-                                "Address of the vault owner."
+                                "Single account ID (CScript or address) or reserved word \"mine\""
```

Comment:
What do you think about: `Vault owner address (or CScript) or reserved word "mine"`. (To not introduce a new terminology account ID)

Reply:
`Single account ID` is already used in `listauctionhistory`, `accounthistorycount` and `listaccounthistory`